### ISSUE TITLE
Update Golang to 1.22 in Dockerfile 

### ISSUE THREAD

**Issue by superjose** [gowebly/gowebly#57]
**Required check list:**

- [x ] I didn't find in the repository's issues section a similar bug.
- [ x] I understand, this is an Open Source and not-for-profit product.
- [ x] This is not about a third-party project, framework, or technology.

**My environment:**
```
Gowebly CLI: v2.3.1 (build with go1.22.0 on darwin/arm64) 
│                                                           
│ Go: 1.22.0 darwin/arm64                                   
│ Node.js version: 20.10.0 (npm 10.2.3)                     
│ Bun version: 1.0.29                                       
│                                                           
│ Is Air tool installed? true                               
│ Is Templ package installed? true   
```
...

**Describe the bug:**

In the main Dockerfile, we need to upgrade Golang to 1.22

...

**Comment by koddr** [comment]
Hi, @superjose 

Thanks! Nice catch.

Fixed in `v2.3.2`.

**Comment by superjose** [comment]
@koddr 
Thanks to you!
I've begun using gowebly and it's been amazing!

For instance, I didn't know Parcel had been upgraded to eslint capabilities! 

Thank you thank you for the CLI!!!
